docs/api/power-monitor.md

```diff
@@ -39,6 +39,14 @@ Emitted when the system is about to lock the screen.
 
 Emitted as soon as the systems screen is unlocked.
 
+### Event: 'user-became-active' _macOS_
+
+Emitted when a login session is activated. See [documentation](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPMultipleUsers/Concepts/UserSwitchNotifications.html#//apple_ref/doc/uid/20002210-CJBJDAGF) for more information.
+
+### Event: 'user-became-inactive' _macOS_
```

Comment:
any reason to name this differently to the native event?

Reply:
@nornagon you mean e.g. `session-did-become-active`? i named it this way mostly bc i felt it was more clear from a user perspective, but am happy to change it.

Reply:
sorry, i specifically meant the native event being "resign-active" rather than "became-inactive".

I don't mind so much using the word `user`, as `session` has another unrelated meaning in Electron and I think it would be more confusing to overload that term than to follow the underlying naming in that respect.

however, given that we aren't writing docs on this, merely exposing the underlying event and pointing at the native docs, i think we should follow the naming in the docs we're linking as closely as possible for minimal confusion.

Reply:
done!